controllers/imagehub/api/v1/image_webhook.go

```diff
@@ -17,69 +17,127 @@ limitations under the License.
 package v1
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"strings"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
 // log is for logging in this package.
 var imagelog = logf.Log.WithName("image-resource")
 
+const saPrefix = "system:serviceaccount:user-system:"
```

Comment:
I need this to replace sa.name.prefix to "" and compare it with org.spec.manager.

Reply:
Since user controller may be deploy to any system namespace, so you need up update this to read ns name from env like: `--env USER_NAMESPACE=xxx`